cnd/src/swap_protocols/rfc003/ethereum/mod.rs

```diff
@@ -82,32 +81,3 @@ impl HtlcParams<Ethereum, asset::Erc20> {
         Erc20Htlc::from(self).into()
     }
 }
-
-impl From<HtlcParams<Ethereum, asset::Ether>> for DeployContract {
```

Comment:
I can't seem to find the difference of the From impl and what you are doing. How does this change contribute to the bugfix?

Reply:
This change helps understanding what is going on by moving all the code converting from htlc to actions in one file. It's a simple move see commit.

Reply:
Didn't realize it is just moved :)

I guess I was just overly thorough in the review because it is a bug fix, hence I was expecting only changes relevant to the bug :)